templates/around/map.html

```diff
@@ -11,6 +11,9 @@
 
 {% block main %}
   <h1>Find someone or something close to <em>you</em></h1>
+  <button id=userCenter onclick=userCenter()>
```

Comment:
you're gonna love it: in html and css, the convention is to use dash (-) or underscode (_).
In this project we use dash.
Therefore the id should be _user-center_ and same in the css ;)